jormungandr/src/fragment/pool.rs

```diff
@@ -562,13 +574,23 @@ pub(super) mod internal {
                 .collect()
         }
 
-        pub fn remove_all<'a>(&mut self, fragment_ids: impl IntoIterator<Item = &'a FragmentId>) {
-            for fragment_id in fragment_ids {
-                let maybe_fragment = self.entries.remove(fragment_id);
-                if let Some(fragment) = maybe_fragment {
-                    self.timeout_queue_remove(&fragment);
-                }
-            }
+        /// returns the ids of updated fragments
```

Comment:
Perhaps adding a doc on what the method actually does before returning would be good too.